src/EnergyPlus/UnitarySystem.cc

```diff
@@ -776,7 +776,6 @@ namespace UnitarySystems {
                 state.dataLoopNodes->Node(this->m_OAMixerNodes[0]).MassFlowRateMin = 0.0;
                 state.dataLoopNodes->Node(this->m_OAMixerNodes[0]).MassFlowRateMinAvail = 0.0;
             }
-            this->m_WSHPRuntimeFrac = 0.0;
```

Comment:
More of the same here. Several data members in parent that can be removed in favor of the data available in the child coils.